heaps/heap_sort.py

```diff
@@ -1,8 +1,23 @@
+from heapq import heappush, heappop
 
-
-def heap_sort(list):
+def heap_sort(lst):
```

Comment:
🤠 Nice work using `heapq`. How could you do it using the `MinHeap` class you implemented?